backend/app/routes.py

```diff
@@ -123,36 +123,63 @@ def get_links_between_two_nodes(from_node_id, to_node_id):
             select_sql = '''
                 WITH results as (
                     SELECT *
-                    FROM pgr_dijkstra(
-                        'SELECT id, source::int, target::int, length::int AS cost FROM here.routing_streets_name',
-                        %(node_start)s,
-                        %(node_end)s
-                    )
+                    FROM here_gis.get_links_btwn_nodes_22_2(
+                        '%(node_start)s',
+                        '%(node_end)s'
+                    ),
+                    UNNEST (links) WITH ORDINALITY AS unnested (link_dir, seq)
                 )
 
                 SELECT 
-                    %(node_start)s,
-                    %(node_end)s,
-                    array_agg(st_name),
-                    array_agg(link_dir),
-                    ST_AsGeoJSON(ST_union(ST_linemerge(geom))) AS geometry
-                FROM results
-                INNER JOIN here.routing_streets_name on edge = id'''
+					link_dir,

```

Comment:
@tankedman can you please clean up this code? Remove commented sections and fix the indenting.

Reply:
Please also add table name prefixes to the selected fields so it's clear where they're coming from.